src/pages/ReviewResults.js

```diff
@@ -41,11 +42,14 @@ export default function ReviewResults(props) {
     let accessElectionReview;
     let rpElectionReview;
 
-    try {
-      accessElection = await Election.findElectionByDarId(darId);
-    } catch (error) {
-      //access election is null, this is expected for a dar with an unreviewed election status
-      //so there is no vote information to display
+    //dars with unreviewed status will not have an election
+    if (isNil(status)) {
+      try {
+        accessElection = await Election.findElectionByDarId(darId);
+      } catch (error) {
+        //access election is null, this is expected for a dar with an unreviewed election status
+        //so there is no vote information to display
+      }
     }
```

Comment:
If we know that `Unreviewed` will have no election, why are we making the query at all? Could you wrap the call with `if (status !== 'Unreviewed')` ?